### ISSUE TITLE
Request for update documentation in org.deeplearning4j.text.documentiterator

### ISSUE THREAD

**Issue by LastRemote** [eclipse/deeplearning4j#2487]
There are some unfinished or deprecated pages in this package, such as LabelledDocument class is almost empty. Even in the org.deeplearning4j.examples.nlp.paragraphvectors package, some functions which are marked deprecated, are not included in the documents. And I found a function getLabels() inside LabelledDocument, which returns a List<String> element, does that mean I would be able to add multiple labels in LabelAwareIterator?
However, in LabelAwareDocumentIterator there is still a function currentLabel() which returns only one String.
And by the way, I am seeking a way to add multiple labels to the same training data. Saying an article is about both physics and biology, for example. Anyone have ideas? I could take the best two of the outputs, but I could not assign two 1s in training process.

**Comment by jageshmaharjan** [comment]
yea, i also suffered from the outdated documentation.

**Comment by agibsonccc** [comment]
On it thanks! Please file any more issues as you see them. This is what the gitter channel is for, rather than spending hours looking at something PLEASE double check with us: https://gitter.im/deeplearning4j/deeplearning4j

**Comment by jageshmaharjan** [comment]
Thank you, the response on gitter is prompt. I am using it.

**Comment by raver119** [comment]
Unfortunately LabelAwareDocumentIterator has nothing to do with LabelledDocument :(

And yes, LabelledDocument pojo/iterators accept as many labels as you want. Idea is simple there: LabelledDocument acts as input data holder, pretty much like DataSet for neural nets. Internally all SequenceVectors algorithms convert it to Sequence<VocabWord>, and transfer labels there as well.

**Comment by tomthetrainer** [comment]
Will add this to a larger document page on Text processing tools and methods.
